UPGRADING

```diff
@@ -19,6 +19,9 @@ PHP 8.2 UPGRADE NOTES
 1. Backward Incompatible Changes
 ========================================
 
+- Core:
+  . Drop support of clang 7 and older.
+
```

Comment:
Do you know what exactly broke clang 7 and older? The patches attempt to feature detect/guard so this wasn't purposefully done on my side.

Reply:
I have referenced this pull requested in #8881, and the issue has been hung for a long time...

Reply:
"clang 7" is insufficiently specific because there are two different compilers called "clang": the one published by llvm.org and the one published by Apple Inc. They have different version numbering schemes and changes that appear in a particular version of one of them may appear in a different version of the other one, or not all.